### PR TITLE
sun.awt.DisableMixing

### PR DESCRIPTION
**Pull request by janakmulani**

### CODE REVIEW

**Comment by janakmulani** [comment]
@AndreasEhret pl merge now

File: core/src/main/java/net/adoptopenjdk/icedteaweb/jnlp/element/security/SecurityDesc.java [hunk 1]
```diff
@@ -232,6 +233,7 @@
             new PropertyPermission(HTTP_KEEP_ALIVE, PROPERTY_RW_ACTION),
             new PropertyPermission(AWT_SYNC_LWREQUESTS, PROPERTY_RW_ACTION),
             new PropertyPermission(AWT_WINDOW_LOCATION_BY_PLATFORM, PROPERTY_RW_ACTION),
+            new PropertyPermission(AWT_DISABLE_MIXING, PROPERTY_RW_ACTION),
```
Comment (sclassen) [inline core/src/main/java/net/adoptopenjdk/icedteaweb/jnlp/element/security/SecurityDesc.java]: The set of properties access is defined in JSR-56.
I don't know if we should change this

File: artifact-all-dependencies/pom.xml [hunk 2]
```diff
@@ -40,7 +40,6 @@
                         </goals>
                         <configuration>
                             <createSourcesJar>true</createSourcesJar>
-                            <shadeSourcesContent>true</shadeSourcesContent>
```
Comment (sclassen) [inline artifact-all-dependencies/pom.xml]: How is this related to this PR?
